sw/device/tests/crypto/aes_gcm_testutils.h

```diff
@@ -0,0 +1,59 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef OPENTITAN_SW_DEVICE_TESTS_CRYPTO_AES_GCM_TESTUTILS_H_
+#define OPENTITAN_SW_DEVICE_TESTS_CRYPTO_AES_GCM_TESTUTILS_H_
+
+#include "sw/device/lib/crypto/drivers/aes.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif  // __cplusplus
+
+typedef struct aes_gcm_test {
+  // Key length.
```

Comment:
nit: doxygen style comments for struct members? (see AES DIF [example](https://cs.opensource.google/opentitan/opentitan/+/master:sw/device/lib/dif/dif_aes.h;drc=59bfc51bb5873a2697035f7c36c77416baaca76f;l=58))